src/http/any-catchall/router.mjs

```diff
@@ -80,24 +83,25 @@ export default async function api (basePath, req) {
       let body = ''
       if (fourOhFour && fourOhFour.includes('.html')) {
         let raw = read(fourOhFour).toString()
-        body = html`${ head(req, status, error) }${ raw }`
+        body = html`${ head({ req, status, error }) }${ raw }`
       }
       else {
-        body = html`${ head(req, status, error) }<page-404 error="${error}"></page-404>`
+        body = html`${ head({ req, status, error }) }<page-404 error="${error}"></page-404>`
       }
       return { status, html: body }
     }
 
     // 200
     const status = state.status || state.code || state.statusCode || 200
     let res = {}
+    const state = Object.assign(initialState, { req, status })
```

Comment:
I'm a little worried that someone might have `status` in their `initialState`.  What about making the object passed to head be `{ req, status, error, store }` where `store` is `state.json`?